src/ui/common/src/components/pages/integration/id/integration-detail-styles.module.css

```diff
@@ -0,0 +1,19 @@
+.icon {    
```

Comment:
i take it ur styling the Autocomplere?

Reply:
I'll do a styled component instead, thanks!

Reply:
Actually, I'm not sure how to make it a styled component or whether we would want to because some of it is a bit specific for this part. Appreciate your feedback about this.
- `.inline-dropdown` - so that I could put the refresh button next to the autocomplete.
- `.icon` - so that I could put the refresh button next to the autocomplete & have it be at the same size.
- `.clickable` / `.not-clickable` - add one or the other class to the refresh button icon depending on whether it is making the post request.

Reply:
@eunice-chan, I think it's typically okay to create a styled component for a single use -- it's typically more readable to have in JSX than to have a separate CSS module. 

Since we're not going to be reusing it widely, I would recommend just putting it in the same file where it's used. If we find a use for it later, we can always refactor it into a reusable component. Does that make sense? 

For an example of where we've done this previously, look at the `GettingStartedTutorial` component, where we have an `InfoCard` styled component that's only used within that file.

Reply:
Sounds good! Would the styled component equivalent for switching between the `.clickable` and `.not-clickable` classes be creating two components and switching between the two?

Reply:
You should be able to use a prop to switch between the two. See: https://mui.com/system/styled/